cobc/field.c

```diff
@@ -581,15 +581,15 @@ cb_build_full_field_reference (struct cb_field* field)
 	cb_tree ref = NULL;
 
 	while (field) {
-		if (!field->flag_filler) {
-			cb_tree rchain = cb_build_reference (field->name);
-			if (ref) {
-				CB_REFERENCE (ref)->chain = rchain;
-			} else {
-				ret = rchain;
-			}
-			ref = rchain;
+
```

Comment:
This is a change of behaviour, the old code goes on with inspecting the parent field when the current one is a filler, the new code here stops the check.

Was that planned?